Kickstarter-iOS/Views/Controllers/ManageViewPledgeRewardReceivedViewController.swift

```diff
@@ -14,5 +14,8 @@ final class ManageViewPledgeRewardReceivedViewController: ToggleViewController {
 
     _ = self.toggle
       |> checkoutSwitchControlStyle
+      |> \.accessibilityLabel %~ {
+        _ in localizedString(key: "Reward_received", defaultValue: "Reward received")
```

Comment:
Are we still waiting for translations or can we use `Strings.Reward_received`? 🤔

Reply:
Tried pulling new strings ... still not translated ... I thought we're not going to block PRs because of localizations but I'm happy to hold on for now

Reply:
I can't remember what we decided about localizations 😭but I think we should at least wait for the key to be available in the `Strings` file?